src/operator/optimizer_op.cc

```diff
@@ -125,8 +152,28 @@ NNVM_REGISTER_OP(rmsprop_update)
 .add_arguments(RMSPropParam::__FIELDS__());
 
 NNVM_REGISTER_OP(rmspropalex_update)
-.describe(R"code(Update function for RMSPropAlex optimizer. The RMSPropAlex code follows the version in
+.describe(R"code(Update function for RMSPropAlex optimizer.
+
+The `RMSPropAlex` is another version of `RMSProp`.
+
+Define :math:`E[g^2]_t` is the decaying average over past squared gradient and
+:math:`E[g]_t` is the decaying average over past gradient.
+
+.. math::
+  E[g^2]_t = \gamma_1 * E[g^2]_{t-1} + (1 - \gamma_1) * g_t^2\\
+  E[g]_t = \gamma_1 * E[g]_{t-1} + (1 - \gamma_1) * g_t\\
+  \Delta_t = \gamma_2 * \Delta_{t-1} - \frac{\eta}{\sqrt{E[g^2]_t - E[g]_t^2 + \epsilon}} g_t\\
+
+The update step is
+
+.. math::
+  \theta_{t+1} = \theta_t + \Delta_t
+
+The RMSPropAlex code follows the version in
 http://arxiv.org/pdf/1308.0850v5.pdf Eq(38) - Eq(45) by Alex Graves, 2013.
+
+Alex suggests the momentum term :math:`\gamma_1` to be 0.95, :math:`\gamma_2`
```

Comment:
Let's not refer to researchers by their first names. Change to "Graves suggests that the momentum term..."